packages/jsii-kernel/lib/api.ts

```diff
@@ -75,13 +80,29 @@ export interface LoadResponse {
 }
 
 export interface CreateRequest {
+  /**
+   * The FQN of the class of which an instance is requested (or "Object")
+   */
   fqn: string;
+  /**
+   * The FQNs of interfaces the instance implements, if any. Declaring
```

Comment:
> Declaring interfaces that the class denoted by `fqn` implements is not necessary.

Maybe describe these as "additional" interfaces then? In addition to the interfaces implemented by the class.

Reply:
Yeah that's how I have started to name them. I'll have to go on the comments & make sure it's homogeneously used.